[KOGITO-816] - Operator provisioning for Task Console

vaibhavjainwiz opened the pull request:
jira issue : https://issues.redhat.com/browse/KOGITO-816

sterobin commented on doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:
```diff
@@ -1186,21 +1186,19 @@ The {PRODUCT} command-line interface (CLI) supports the following operations on
 |`{PRODUCT_INIT} install explainability --infra __KAFKA_INFRA_NAME__`
 |`{PRODUCT_INIT} install explainability --infra kogito-kafka-infra`
 
-|Install {PRODUCT} supporting services or infrastructure components for a specified {PRODUCT} project. Use this syntax if you did not use the `new-project` or `use-project` commands to connect the {PRODUCT} Operator to a specified project.
```
@vaibhavjainwiz This undoes the change the @Kaitou786 made to incorporate the Kogito supporting service. Unclear here. @Kaitou786 ?

Kaitou786 replied:
Yes it is correct :)

sterobin commented:
@ricardozanini @Kaitou786 @vaibhavjainwiz I'm realizing there KogitoSupportingServices change impacts all console sections in the docs and requires new screenshots, but I need a working OCP web console environment for that, and we need to agree on the Kogito CLI commands. @Kaitou786  merged a PR with one set of CLI commands, but @vaibhavjainwiz is merging a different set now, so I'm confused. Pinging the Kogito #Cloud channel for your help.

sterobin commented on doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:
```diff
@@ -1186,21 +1186,21 @@ The {PRODUCT} command-line interface (CLI) supports the following operations on
-
-`{PRODUCT_INIT} install data-index --infra kogito-infinispan-infra --infra kogito-kafka-infra -p kogito-travel-agency`
-
-`{PRODUCT_INIT} install jobs-service --infra kogito-infinispan-infra --infra kogito-kafka-infra -p kogito-travel-agency`
```
@vaibhavjainwiz Why remove the mgmt-console and jobs-service lines? Those are still accurate too, correct? The purpose was to illustrate how to specify the infra for each of these during installation.

vaibhavjainwiz replied:
Already shown in their respective services example

sterobin replied:
Sure, okay. Thanks then.

vaibhavjainwiz commented:
https://github.com/kiegroup/kie-docs/pull/3008#discussion_r516029714

I had updated this text with intend to make it more clear but it turn out to be worst. I had reverted it

sterobin reviewed (approved):
@vaibhavjainwiz I pushed a couple tweaks. I agree with what you said about duplication in the supporting services table row, makes sense. I included the commented-out portion about getting the task console up in OCP, and I'll update the screenshot once I get that either from you or I can get a web console environment set up. I need to do the same for many other web console screenshots.

Also, I removed the section "Operator interaction with the Task Console" because we don't need a section like that for every console or supporting service, only the ones that merit a bit more elaboration for additional info or options regarding how the Operator works with it. In this case, everything there is covered in the existing section about using the Task Console, so we're good.

Thanks!

sterobin commented:
@vaibhavjainwiz I updated the screenshot, so good to merge once others approve. I'll do the larger update to all other screenshots separately.

@ricardozanini @sutaakar Down to you.
